public/data/kumaran.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "T.kumaran",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/synackred.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/synackred",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/kumaran923",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing,Pentesting,BugbountyHunting",
```

Comment:
I think you missed the space between Bugbounty and Hunting, I've also added spaces to the rest as a **suggestion**

```suggestion
        "title": "Started Freelancing, Pentesting, Bugbounty Hunting",
```